frontend/src/js/external-forms/transformQueryToApi.ts

```diff
@@ -2,11 +2,15 @@ import { transformElementsToApi } from "../api/apiHelper";
 import { Form } from "./config-types";
 
 function transformElementGroupsToApi(elementGroups) {
-  return elementGroups.map(({ concepts, connector, ...rest }) => ({
-    type: connector,
-    children: transformElementsToApi(concepts),
-    ...rest,
-  }));
+  return elementGroups.map(({ concepts, connector, ...rest }) =>
+    concepts.length > 1
+      ? {
+          type: connector,
+          children: transformElementsToApi(concepts),
+          ...rest,
+        }
+      : { ...transformElementsToApi(concepts)[0], ...rest }
+  );
 }
```

Comment:
Macht das hier den wrap in ein and oder ein or?

Reply:
es sieht so aus als würde er in Zeile 6 entscheiden ob er es in AND/OR packen will und in Zeile 12 direkt platziert